django_teams/templates/django_teams/_table.html

```diff
@@ -0,0 +1,31 @@
+{% load teams %}
+<tr>
+    <td>
+      <a href="{% url 'team-detail' pk=object.id %}">{{ object }}</a>
+      {% if not user.is_anonymous %}
+          {% if object.role == 20 %}
+              (<a href="{% url 'team-edit' pk=object.pk %}">Admin</a>)
+          {% endif %}
+      {% endif %}
+    </td>
+    <td>{% if object.description %}{{ object.description|truncatewords:15 }}{% endif %}</td>
+    <td>
+        {{ object.owner }}
+    </td>
+    <td>{{ object.member_count }}</td>
```

Comment:
In correctly counts members for classes of which you are the owner (maybe member too?).